arch/arm/src/armv7-a/arm_addrenv.c

```diff
@@ -778,22 +671,18 @@ int up_addrenv_clone(const arch_addrenv_t *src,
  *   is created that needs to share the address environment of its task
  *   group.
  *
- *   NOTE: In some platforms, nothing will need to be done in this case.
- *   Simply being a member of the group that has the address environment
- *   may be sufficient.
- *
  * Input Parameters:
- *   group - The task group to which the new thread belongs.
- *   tcb   - The TCB of the thread needing the address environment.
+ *   ptcb  - The tcb of the parent task.
+ *   tcb   - The tcb of the thread needing the address environment.
  *
  * Returned Value:
  *   Zero (OK) on success; a negated errno value on failure.
  *
  ****************************************************************************/
 
-int up_addrenv_attach(struct task_group_s *group, struct tcb_s *tcb)
+int up_addrenv_attach(struct tcb_s *ptcb, struct tcb_s *tcb)
```

Comment:
should we remove ptcb?

Reply:
I'm not sure what this function is supposed to do. It is implemented as blank by all 3 platforms where it exists.

However, to keep the meaning of the original function ptcb is needed, "attach" means to join the parent's address environment.